peer/chaincode/common.go

```diff
@@ -180,6 +180,14 @@ func chaincodeInvokeOrQuery(cmd *cobra.Command, invoke bool, cf *ChaincodeCmdFac
 	return nil
 }
 
+// CollectionType enumerates the various types of private data collections.
```

Comment:
I don't this this change should be part of this commit since it references transient data.